app/src/main/res/values/strings.xml

```diff
@@ -596,5 +596,39 @@
     <string name="new_accounts_mixed_unmixed">New accounts will be created for mixed and unmixed.</string>
     <string name="new_account_mixed">New account will be created for mixed.</string>
     <string name="new_account_umixed">New account will be created for unmixed</string>
+    <string name="new_caps">NEW</string>
+    <string name="privacy_is_here">Privacy is here</string>
+    <string name="privacy_intro_desc">You can now use CoinShuffle++ mixer to mix your DCRs, right in the app.</string>
+    <string name="dismiss">Dismiss</string>
+    <string name="setup_mixer">Set up mixer</string>
+    <string name="privacy_popup_message">Set up CoinShuffle++ mixer in Privacy</string>
+    <string name="see_all">See all</string>
+    <string name="privacy_intro_title">How does CoinShuffle++ mixer enhance your privacy?</string>
+    <string name="privacy_intro_desc_1">CoinShuffle++ mixer can mix your DCRs through CoinJoin transactions.</string>
+    <string name="privacy_intro_desc_2">Using mixed DCRs protects you from exposing your financial activities to the public (e.g. how much you own, who pays you).</string>
+    <string name="setup_mixer_btn_title">Set up mixer for this wallet</string>
+    <string name="privacy_intro_dialog_title">Set up mixer by creating two needed accounts</string>
+    <string name="privacy_intro_dialog_desc"><![CDATA[Two dedicated accounts (“mixed” & “unmixed”) will be created in order to use the mixer.<br/><br/><b><font color="#091440">This action cannot be undone.</font></b>]]></string>
+    <string name="begin_setup">Begin setup</string>
+    <string name="mixer_setup_completed">Mixer setup completed</string>
+    <string name="ready_to_mix">Ready to mix</string>
+    <string name="unmixed_balance">Unmixed balance</string>
+    <string name="mixed_balance">Mixed balance</string>
+    <string name="mixer_settings">Mixer Settings</string>
+    <string name="account_branch">Account branch</string>
+    <string name="shuffle_port">Shuffle port</string>
+    <string name="shuffle_server">Shuffle server</string>
+    <string name="keep_app_opened">Keep this app opened</string>
+    <string name="mixer_help_title">How to use the mixer?</string>
+    <string name="mixer_help_desc"><![CDATA[When you turn on the mixer, your unmixed DCRs in this wallet (unmixed balance) will be gradually mixed.<br/><br/><b><font color="#091440">Important: keep this app opened while mixer is running.</font></b><br/><br/>Mixer will automatically stop when unmixed balance are fully mixed.]]></string>
+    <string name="mixer_is_running">Mixer is running…</string>
+    <string name="keep_this_app_opened">Keep this app opened</string>
+    <string name="mixer_status_info">Mixer will automatically stop when unmixed balance are fully mixed.</string>
+    <string name="check_mixer_status">Check mixer status</string>
+    <string name="mixing_elp">Mixing…</string>
```

Comment:
is this meant to spell `mixing_help`? or is `elp` short for something?

Reply:
I abbreviated `ellipsis` as `elp`.